public/doc/swagger-2.yaml

```diff
@@ -843,12 +865,18 @@ definitions:
     type: object
     required:
       - name
-      - template
     properties:
+      id:
+        type: string
+        readOnly: true
       name:
         type: string
       description:
         type: string
+      groups:
```

Comment:
You can add `group_ids` which is an array of ids. This is helpful to create a new workflow.